jwst/resample/resample.py

```diff
@@ -374,6 +374,29 @@ def update_exposure_times(self, output_model):
         output_model.meta.exposure.end_time = max(exposure_times['end'])
         output_model.meta.resample.product_exposure_time = total_exposure_time
 
+        # Update other exposure time keywords:
+        # XPOSURE (identical to EFFEXPTM)
+        xposure = output_model.meta.exposure.frame_time * (output_model.meta.exposure.ngroups *\
```

Comment:
Given that the `XPOSURE` keyword contains identical values as `EFFEXPTM` and the combined value for the latter is already computed as `total_exposure_time`, then why not just set `xposure = total_exposure_time`, instead of trying to recompute it all from scratch here. Also, there's apparently something in this recomputation that doesn't work right, because it's killing all the CI tests.